htlcswitch/interfaces.go

```diff
@@ -100,22 +100,21 @@ type ChannelLink interface {
 	// policy to govern if it an incoming HTLC should be forwarded or not.
 	UpdateForwardingPolicy(ForwardingPolicy)
 
-	// HtlcSatifiesPolicy should return a nil error if the passed HTLC
-	// details satisfy the current forwarding policy fo the target link.
-	// Otherwise, a valid protocol failure message should be returned in
-	// order to signal to the source of the HTLC, the policy consistency
-	// issue.
-	HtlcSatifiesPolicy(payHash [32]byte, incomingAmt lnwire.MilliSatoshi,
+	// CanForwardHtlc should return a nil error if the passed HTLC details
+	// satisfy the current forwarding policy fo the target link. Otherwise,
+	// a valid protocol failure message should be returned in order to
+	// signal to the source of the HTLC, the policy consistency issue.
+	CanForwardHtlc(payHash [32]byte, incomingAmt lnwire.MilliSatoshi,
 		amtToForward lnwire.MilliSatoshi,
 		incomingTimeout, outgoingTimeout uint32,
 		heightNow uint32) lnwire.FailureMessage
 
-	// HtlcSatifiesPolicyLocal should return a nil error if the passed HTLC
-	// details satisfy the current channel policy.  Otherwise, a valid
-	// protocol failure message should be returned in order to signal the
-	// violation. This call is intended to be used for locally initiated
-	// payments for which there is no corresponding incoming htlc.
-	HtlcSatifiesPolicyLocal(payHash [32]byte, amt lnwire.MilliSatoshi,
+	// CanSendHtlc should return a nil error if the passed HTLC details
+	// satisfy the current channel policy.  Otherwise, a valid protocol
+	// failure message should be returned in order to signal the violation.
+	// This call is intended to be used for locally initiated payments for
+	// which there is no corresponding incoming htlc.
+	CanSendHtlc(payHash [32]byte, amt lnwire.MilliSatoshi,
```

Comment:
Send -> Carry? As it's asking if this link can serve as transit for the outgoing HTLC (it's "hop zero").

Reply:
And this one CheckHtlcTransit